tests/test_reader.py

```diff
@@ -41,3 +43,27 @@ def test_read(self):
 
 		unknown2 = '{"type":"Person", "fishbat": "bob"}'
 		self.assertRaises(DataError, self.reader.read, unknown)
+
+	def test_attrib_assign(self):
+		vocab.add_attribute_assignment_check()
+
+		data = """
+		{
+		  "id": "https://linked.art/example/activity/12", 
+		  "type": "AttributeAssignment", 
+		  "assigned": {
+		    "id": "https://linked.art/example/name/10", 
+			"type": "Name", 
+		    "content": "Exhibition Specific Name"
+		  }, 
+		  "assigned_property": "identified_by", 
+		  "assigned_to": {
+		    "id": "https://linked.art/example/object/12", 
+		    "type": "HumanMadeObject", 
+		    "_label": "Real Painting Name"
+		  }
+		}
+		"""
+		d = self.reader.read(data)
+		
+
```

Comment:
This test doesn't seem to have any assertions. Was it meant to be filled in?

Reply:
Oh, heh.  I just had it do the read and fixed errors... it probably should have an assert or two as well.

Reply:
OK. Modulo this missing test, everything looks good.